src/chrome/content/rules/LEGO.xml

```diff
@@ -1,141 +1,62 @@
 	<target host="shop.education.lego.com" />
+	<target host="ideas.lego.com" />
+	<target host="jobsearch.lego.com" />
+	<target host="lan.lego.com" />
 	<target host="moderation.lego.com" />
-	<target host="rebrick.lego.com" />
+	<target host="ninjagochallenge.lego.com" />
+	<target host="savethecity.lego.com" />
+	<target host="services.lego.com" />
 	<target host="shop.lego.com" />
+	<target host="www.us.lego.com" />
+	<target host="wwwsecure.us.lego.com" />
+	<target host="usaevents.lego.com" />
+	<target host="worldofcreativity.lego.com" />
 	<target host="wwwsecure.lego.com" />
 
-	<!--	Complications:
-				-->
-	<!--target host="www.lego.com" /-->
-
-		<exclusion pattern="^http://club\.lego\.com/(?!en-us/join/magazinesubscription)" />
-
-			<!--	+ve:
-					-->
-			<test url="http://club.lego.com/de-de/" />
-
-			<!--	-ve:
-					-->
-			<test url="http://club.lego.com/en-us/join/magazinesubscription" />
-
-		<exclusion pattern="^http://education\.lego\.com/(?!\w\w-\w\w/[\w-]+/|Design/|WebResource\.axd)" />
-
-			<!--	+ve:
-					-->
-			<test url="http://education.lego.com/downloads/" />
-			<test url="http://education.lego.com/ja-jp" />
-			<test url="http://education.lego.com/ko-kr" />
-			<test url="http://education.lego.com/zh-cn" />
-
-			<!--	-ve:
-					-->
-			<test url="http://education.lego.com/ko-kr/products/" />
-
-		<exclusion pattern="^http://shop.lego.com/(?!VIP/modal/vipLearnMoreModal\.jsp)" />
-
-			<!--	+ve:
-					-->
-			<test url="http://shop.lego.com/VIP" />
-
-			<!--	-ve:
-					-->
-			<test url="http://shop.lego.com/VIP/modal/vipLearnMoreModal.jsp" />
-
-		<!--	Breakage is more extensive than this:
-				https://github.com/EFForg/https-everywhere/issues/2858
-
-			So default-inclusion of www → wwwsecure is disabled in favor of
-			including what we know to work as we find it.
-
-			Exclusions and tests are left as commentary and for future use.
-
-			404:
-				-->
-		<!--exclusion pattern="^http://www\.lego\.com/(?:\w\w-\w\w/bionicle|friends)(?:$|[?/])" /-->
-
-			<!--	+ve:
-					-->
-			<!--test url="http://www.lego.com/en-us/bionicle" /-->
-			<!--test url="http://www.lego.com/en-us/bionicle/" /-->
-			<!--test url="http://www.lego.com/en-us/bionicle/?domainredir=" /-->
-			<!--test url="http://www.lego.com/en-us/bionicle/?domainredir=www.bionicle.com" /-->
-			<!--test url="http://www.lego.com/en-us/friends" /-->
-
-
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^\.lego\.com$" name="^XLB$" /-->
-
-	<securecookie host="^(?:aboutus|account2?|services\.account|assets|rebrick)\.lego\.com$" name=".+" />
-
+	<target host="lc-www-live-s.legocdn.com" />
+	<test url="http://lc-www-live-s.legocdn.com/r/www/r/globalnavigationservices/content/images/lego-logo-menu.svg" />
 
-	<!--rule from="^http://www\.lego\.com/"
-		to="https://wwwsecure.lego.com/" /-->
+	<!-- 403 -->
+	<exclusion pattern="^http://jobsearch\.lego\.com/$" />
+	<test url="http://jobsearch.lego.com/sap/bc/webdynpro/sap/" />
```

Comment:
Can we put this exclusion with the target? I literally had to delete my previous review when I saw it.

Reply:
Do you mean the sorting?

Reply:
yes

Reply:
Sure, done.